frontend/src/scenes/funnels/FunnelLineGraph.tsx

```diff
@@ -20,29 +21,36 @@ export function FunnelLineGraph({
     return (
         <LineGraph
             data-attr="trend-line-graph-funnel"
-            type="line"
+            type={GraphTypes.Line}
             color={color}
-            datasets={steps}
+            datasets={steps as unknown as GraphDataset[]}
```

Comment:
Is there any way to have a better cast here? `as unknown as` is sketchy :).

Reply:
Spent some time on this but it looks like it might be kind of a time sink since `funnel` datasets have very different shapes from normal trend datasets. I think it might be better to leave a TODO comment here for a later PR.